src/main/java/edu/cornell/kfs/concur/ConcurUtils.java

```diff
@@ -16,7 +16,7 @@ public static boolean isTravelRequestURI(String URI) {
     }
 
     public static String extractKFSInfoFromConcurString(String concurString) {
-        String kfsInfo = StringUtils.EMPTY;
+        String kfsInfo = concurString;
         if (doesStringContainOpenCloseParenthesis(concurString)) {
             kfsInfo = concurString.substring(concurString.lastIndexOf(CUKFSConstants.LEFT_PARENTHESIS) + 1, concurString.lastIndexOf(CUKFSConstants.RIGHT_PARENTHESIS));
```

Comment:
That is a good point. I think this could be cleaned up to be more clear. Some fields in the concur xml report details contain the value in the format: "(code) description". In this case we only care about the code so we extract only that. In other cases it just contains a value and we pull the whole value in. I think it would be better to separate the two based on specific fields instead of trying to treat all values with the same utility function. I will update the code and try to make it more clear. Thank you.